addon/index.ts

```diff
@@ -117,7 +115,7 @@ async function setupHashSupport(router: EmberRouter) {
 
 const CACHE = new WeakMap<ApplicationInstance, MutationObserver>();
 
-async function eventuallyTryScrollingTo(owner: ApplicationInstance, url?: string) {
+function eventuallyTryScrollingTo(owner: ApplicationInstance, url?: string) {
```

Comment:
Does this need to be called _eventually_ try scrolling if there is no longer a delay?

Reply:
Probably not - I was trying to minimize changes to the original project, but if people feel strongly about it, I can rename.